packages/loader/driver-utils/src/test/blobAggregation.spec.ts

```diff
@@ -62,7 +62,7 @@ class InMemoryStorage {
     async readBlob(id: string) {
         const blob = this.blobs.get(id);
         if (blob !== undefined) {
```

Comment:
Not that it matters a lot, but code becomes more streamlined if you do this:
```
assert(blob !== undefined, "unknown blob id");
return blob;
```
here and in all other places like that

Reply:
i see. it doesn't seem there are other places like this with an error message. other places are more like if blob is undefined do this, else do that